bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2.py

```diff
@@ -14,7 +14,7 @@
 
 
 from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
-from google.cloud.bigquery_datatransfer_v1.proto import transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
+from google.cloud.bigquery.datatransfer_v1.proto import transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
```

Comment:
The underscore is crucial, here.

Reply:
@lukesneeringer @landrito @eoogbe 
Is this file a protoc artifact? Do we have any influence over the output of whatever tool makes this file?